File: Szhchem.Data/Models/Property.cs
using MongoDB.Bson;
using MongoDB.Bson.Serialization.Attributes;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Szhchem.Data
{
    public class Property
    {
        [BsonId]
        [BsonRepresentation(BsonType.ObjectId)]
        public string PropertyId { get; set; }
        public long SubstanceID { get; set; }
        public long MolID { get; set; }
        public int LanguageCode { get; set; }
        public int PropertyCategory { get; set; }
        public int PropertyCode { get; set; }
        public string PropertyValue { get; set; }
        public string PropertyReference { get; set; }
    }    
}

File: molfinder/ClientApp/src/models/Property.ts
export default class Property {
    public id: string;
    public substanceID: number;
    public molID: number;
    public languageCode: number;
    public propertyCategory: number;
    public propertyCode: number;
    public propertyValue: string;
    public propertyReference: string;

    constructor(id: string, substanceID: number, molID: number, languageCode: number, propertyCategory: number, propertyCode: number, propertyValue: string, propertyReference: string) {
        this.id = id;
        this.substanceID = substanceID;
        this.molID = molID;
        this.languageCode = languageCode;
        this.propertyCategory = propertyCategory;
        this.propertyCode = propertyCode;
        this.propertyValue = propertyValue;
        this.propertyReference = propertyReference;
    }
}